tool/travis.sh

```diff
@@ -38,6 +38,9 @@ else
   # Run some validations on the repo code.
   ./bin/plugin lint
 
+  # Check plugin-referenced urls for liveness.
+  dart tool/grind.dart check-urls
```

Comment:
These urls may be flaky outside of our control; perhaps as future work, move the check into a new travis job (so that when it fails, its easier to de-flake from the rest of the build).

Reply:
My thinking too.  I'll follow up with another PR that adds a bot.